main.py

```diff
@@ -96,7 +118,9 @@ def main(config):
             configs = [config]
 
         print(f"Submitting {len(configs)} jobs")
-        executor = submitit.AutoExecutor(folder=args.logdir / "%j")
+        executor = submitit.AutoExecutor(
+            folder=args.logdir / "%j", slurm_max_num_timeout=3
```

Comment:
I don't imagine using more than 3 requeues for most use cases, but did we want to make this a command line argument like the other slurm options (default=3)?

Reply:
Probably not at this point; I don't think it'll be used. We can switch if we notice otherwise in our workflow.